distribution/src/main/release/samples/jaxws_spring_boot/pom.xml

```diff
@@ -34,6 +35,15 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
The `spring-boot-starter-actuator` is missed, sample project fails on startup (no registry):
```
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-actuator</artifactId>
        </dependency>
```

Reply:
You are right, I forgot to update this sample after I modified poms.